File: classes/parte-4-control-del-programa/061-switch-case-y-fallthrough/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	d, _ := strconv.Atoi(strings.TrimSpace(line))
	var dia string
	switch d {
	case 1:
		dia = "lunes"
	case 2:
		dia = "martes"
	case 3:
		dia = "miercoles"
	case 4:
		dia = "jueves"
	case 5:
		dia = "viernes"
	case 6:
		dia = "sabado"
	case 7:
		dia = "domingo"
	default:
		dia = "invalido"
	}
	fmt.Printf("dia=%s\n", dia)
}

File: classes/parte-4-control-del-programa/061-switch-case-y-fallthrough/implementaciones/typescript/main.ts
import { readFileSync } from "node:fs";

const d: number = parseInt(readFileSync(0, "utf8").trim(), 10);
let dia: string;
switch (d) {
  case 1: dia = "lunes"; break;
  case 2: dia = "martes"; break;
  case 3: dia = "miercoles"; break;
  case 4: dia = "jueves"; break;
  case 5: dia = "viernes"; break;
  case 6: dia = "sabado"; break;
  case 7: dia = "domingo"; break;
  default: dia = "invalido";
}
console.log(`dia=${dia}`);
